src/main/java/org/interledger/codecs/oer/ipr/InterledgerPaymentRequestOerCodec.java

```diff
@@ -0,0 +1,52 @@
+package org.interledger.codecs.oer.ipr;
+
+import org.interledger.Condition;
+import org.interledger.InterledgerAddress;
+import org.interledger.codecs.Codec;
+import org.interledger.codecs.CodecContext;
+import org.interledger.codecs.InterledgerPaymentRequestCodec;
+import org.interledger.codecs.oer.OerUint8Codec.OerUint8;
+import org.interledger.ilp.InterledgerPayment;
+import org.interledger.ipr.InterledgerPaymentRequest;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.util.Objects;
+
+/**
+ * An implementation of {@link Codec} that reads and writes instances of {@link InterledgerAddress}.
```

Comment:
`InterledgerAddress` should be renamed to `InterledgerPaymentRequest` in the javadoc.